src/Writer/Common/Entity/Sheet.php

```diff
@@ -160,4 +164,45 @@ public function getAutoFilter(): ?AutoFilter
     {
         return $this->autoFilter;
     }
+
+    /**
+     * @param positive-int ...$columns One or more columns with this width
+     */
+    final public function setColumnWidth(float $width, int ...$columns): void
```

Comment:
Overall this PR looks good, but I don't like the complexity under the hood of this function signature.
I'd prefer force the user call this multiple times more than force ourself to maintain such a complex function content.
```suggestion
    final public function setColumnWidth(float $width, int $column): void
```

Reply:
Thanks for reviewing!

This decision was based on an existing method: https://github.com/openspout/openspout/blob/90ceeefef3750dd2afab95689ed2107dd576905e/src/Writer/Common/AbstractOptions.php#L30 Do you want me to deviate from that?

Reply:
Didn't remember that (aweful) code, that's for pointing it out